src/orchard/java/jdk11.clj

```diff
@@ -0,0 +1,7 @@
+(ns orchard.java.jdk11
```

Comment:
I doubt there will be anything else here that's not related to modules, so this can also be something like `orchard.java.modules`. I'm not very fond of linking ns names to specific versions of something. But I don't feel strongly about this.